backend/app/api/api_v1/routers/admin.py

```diff
@@ -276,6 +253,43 @@ async def import_law_policy(
         ) from e
 
 
+def start_import(db, s3_client, document_create_objects):
+    document_parser_inputs: list[DocumentParserInput] = []
+    documents_ids_already_exist: list[str] = []
```

Comment:
we don't need to calculate this again, and the return value won't be used anyway

Reply:
I didn't check this - thanks :face_exhaling: